build.gradle.kts

```diff
@@ -67,17 +67,11 @@ publishing {
     publications.configureEach {
         if (this is MavenPublication) {
             val libraryType = name
-            artifactId = when (libraryType) {
```

Comment:
So, how will be named different artefacts then? For example, for `js`? I'm not sure that all of them should have one name.

Reply:
They won't. Publications have corect names (postfixed by target platform) by default:

![image](https://github.com/Scogun/kcron-common/assets/9965210/d5c44a1e-6124-4641-a2e1-9e04bf8e5aaa)

Reply:
So, we will have `kcron-common-jvm` instead of just `kcron-common` and exact `kcron-common` instead of `kcron-common-native` one. Maybe it makes sense. However, let's keep `-common` postfix and in this case the version could be increased only up to `0.6.1`.